src/scripts/charts/AxisChart.js

```diff
@@ -362,8 +362,8 @@ export default class AxisChart extends BaseChart {
 		this.updating = true;
 
 		this.y_sums = new Array(this.x_axis_positions.length).fill(0);
-		this.y.map(d => {
-			d.values.map( (value, i) => {
+		this.y.forEach(d => {
+			d.values.forEach( (value, i) => {
```

Comment:
`reduce`

Reply:
Now sure how a reduce might work here. Would the whole calculation have to be changed?

Reply:
now that I checked again, it's not as simple as others so, better to keep it like this.